### PR TITLE
Feature: Added MolarMass Quantity Type

### LINKED ISSUE #303: Request: Molar Mass
together with my colleague @ferittuncer, we are modelling a thermodynamic library for a several chemical species. To be able to freely change unit frame, we need to have a `MolarMass `quantity type data on hands. 

For this purpose we propose to contribute to the library by including this type. 

Waiting for feedbacks. Best regards

### PR DESCRIPTION
**Pull request by ferittuncer**
Implemented molar mass which we discussed in #303 

Fixes #303

### CODE REVIEW

File: UnitsNet/UnitDefinitions/MolarMass.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "г/моль" ],
+          "AbbreviationsWithPrefixes": [ "нг/моль", "мкг/моль", "мг/моль", "сг/моль", "дг/моль", "даг/моль", "гг/моль", "кг/моль" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerMole",
+      "PluralName": "PoundsPerMoles",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/MolarMass.json]: I'm fixing this to `PoundsPerMole` and `GramsPerMole`.

File: UnitsNet/UnitDefinitions/MolarMass.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      "SingularName": "PoundPerMole",
+      "PluralName": "PoundsPerMole",
+      "FromUnitToBaseFunc": "x*0.45359237",
+      "FromBaseToUnitFunc": "x/0.45359237",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/mol" ],
+          "AbbreviationsWithPrefixes": [ "KLbs/mol", "MLbs/mol" ]
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/MolarMass.json]: Kilo should be lowercase `klbs`. Also, I think I prefer the singular form when it comes to abbreviations. I know there are varying practices here, but unless `klbs/mol` is clearly preferred over `klb/mol` in this domain then I think we should use singular form to stay consistent in the library.

**Comment by angularsen** [comment]
I took the liberty of editing my suggestions in directly. Please review if you agree, then this should be ready to merge.

**Comment by angularsen** [comment]
I can't push the generated code apparently, so if you could run `generate-code.bat` and push the regenerated code up that would be great.

**Comment by ferittuncer** [comment]
I reviewed your changes and didn't find any problem. I think it's ready to merge.

File: UnitsNet/UnitDefinitions/MolarMass.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Deca", "Hecto", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/mol" ]
+        },
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "г/моль" ],
+          "AbbreviationsWithPrefixes": [ "нг/моль", "мкг/моль", "мг/моль", "сг/моль", "дг/моль", "даг/моль", "гг/моль", "кг/моль" ]
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/MolarMass.json]: Can we drop these prefixes and rely on the auto-generated prefixes, like the unit below?
It seems to me they only add the standard prefix letter here.

Comment (angularsen) [reply UnitsNet/UnitDefinitions/MolarMass.json]: Never mind, we can do that separately. No need to delay the PR over this.

**Comment by angularsen** [comment]
Awesome, looks good to me!

File: UnitsNet/UnitDefinitions/MolarMass.json (hunk 3 above)
Comment (ferittuncer) [reply UnitsNet/UnitDefinitions/MolarMass.json]: Yes, I thought those prefixes were necessary. But if it's not, let's drop them.

Comment (angularsen) [reply UnitsNet/UnitDefinitions/MolarMass.json]: Actually, it turns out only English is supported by the scripts. It should be trivial to extend the script to support other languages too as long as they simply prepend a certain string for each prefix. Russian seems to do this, but it is not a language I am familiar with.

Comment (ferittuncer) [reply UnitsNet/UnitDefinitions/MolarMass.json]: I am going to open an issue about this to remember later.

**Pull request merged**